util/build_consts.sh

```diff
@@ -25,8 +29,14 @@ else
   echo "Non-bash shells are currently not supported." >&2
   exit 1
 fi
-BUILD_ROOT="${BUILD_ROOT:-"$REPO_TOP"}"
 
+if git describe --always >/dev/null 2>/dev/null; then
```

Comment:
nit: Don't re-execute `git describe --always`, just capture the value and `-z` it, or check `$?`.